lib/tasks/import_events.rake

```diff
@@ -51,6 +51,19 @@ namespace :import do
           Rails.logger.info "Cannot import! #{event.title}"
         end
       end
+
+      list_of_children = csv_event_row[event_hash[:organisers]]&.strip
```

Comment:
I would just use something on the line of `organiser` rather than `children` for these variables, so it's clearer what relationship we are adding.

Other than that, it looks good :)

Reply:
Thanks @Levia you are right it is much clearer now that I changed the variable names!